dbt/adapters/sqlserver/connections.py

```diff
@@ -49,7 +49,8 @@ class SQLServerCredentials(Credentials):
     # "sql", "ActiveDirectoryPassword" or "ActiveDirectoryInteractive", or
     # "ServicePrincipal"
     authentication: Optional[str] = "sql"
-    encrypt: Optional[str] = "yes"
+    encrypt: Optional[bool] = False
```

Comment:
by "breaks old way of setting up profile" do you mean that I've toggled Encrypt to now be `False` by default instead of `True`?

Reply:
@mikaelene

Reply:
Something like that. I removed the cert settings stuff from my profile and it stopped working. Just have to make sure the pr doesn’t break the adapter for all users 😀